cypress/e2e/marc/marc-authority/reporting-marc-authority/marc-authority-reporting-not-include-data-deleted-record.cy.js

```diff
@@ -0,0 +1,206 @@
+      MarcAuthoritiesSearch.selectAuthorityByIndex(0);
+      MarcAuthority.edit();
+      QuickMarcEditor.waitLoading();
+
+      cy.wait(2000);
+      QuickMarcEditor.updateExistingField(testData.tag100, testData.updatedTag100Value1);
+      QuickMarcEditor.saveAndCloseUpdatedLinkedBibField();
+      QuickMarcEditor.confirmUpdateLinkedBibsKeepEditing(1);
+
+      MarcAuthorities.searchBy(testData.searchOption, testData.authorityHeading2);
+      MarcAuthoritiesSearch.selectAuthorityByIndex(0);
+      MarcAuthority.edit();
+      QuickMarcEditor.waitLoading();
+
+      cy.wait(2000);
+      QuickMarcEditor.updateExistingField(testData.tag100, testData.updatedTag100Value2);
+      QuickMarcEditor.pressSaveAndClose();
+      MarcAuthority.delete();
+      QuickMarcEditor.confirmDeletingRecord();
+
+      MarcAuthorities.clickActionsAndReportsButtons();
+      MarcAuthorities.fillReportModal(today, tomorrow);
+      MarcAuthorities.clickExportButton();
+
+      cy.intercept('POST', '/data-export-spring/jobs').as('getId');
+      cy.wait('@getId', { timeout: 10000 }).then((item) => {
+        const jobID = item.response.body.name;
+        const expectedJobData = [
+          jobID,
+          expectedJobValues.status,
+          expectedJobValues.jobType,
+          expectedJobValues.description,
+          testData.userProperties.username,
+          todayWithoutPaddingZero,
+        ];
+        const expectedJobDetails = {
+          jobID,
+          status: expectedJobValues.status,
+          jobType: expectedJobValues.jobType,
+          outputType: expectedJobValues.outputType,
+          description: expectedJobValues.description,
+          source: testData.userProperties.username,
+          startDate: todayWithoutPaddingZero,
+          endDate: todayWithoutPaddingZero,
+        };
+        MarcAuthorities.checkCalloutAfterExport(jobID);
+        cy.visit(TopMenu.exportManagerPath);
+        ExportManagerSearchPane.waitLoading();
+        ExportManagerSearchPane.searchByAuthorityControl();
+        ExportManagerSearchPane.verifyJobDataInResults(expectedJobData);
+        ExportManagerSearchPane.verifyResultAndClick(jobID);
+        ExportManagerSearchPane.verifyJobDataInDetailView(expectedJobDetails);
+        ExportManagerSearchPane.downloadLastCreatedJob(item.response.body.name);
+      });
+
+      const downloadedReportDate = DateTools.getFormattedDate({ date: new Date() });
+      const fileNameMask = `${downloadedReportDate}*`;
+      FileManager.verifyFile(
+        MarcAuthorities.verifyMARCAuthorityFileName,
+        fileNameMask,
+        MarcAuthorities.verifyContentOfExportFile,
+        [
+          'Last updated',
+          `${downloadedReportDate}`,
+          'Original heading',
+          testData.authorityHeading1,
+          'New heading',
+          testData.authorityHeading1,
+          'Identifier',
+          testData.authority001FieldValue,
+        ],
+      );
```

Comment:
Delete the created folder after you finish testing, FileManager.deleteFolder(Cypress.config('downloadsFolder'));